source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -54,82 +54,83 @@ The following figure depicts the import process of an OVA that was generated by
+The user will be able to select a VM (or a template) to be exported as an OVA file. In such cases, the user needs to specify a path on a host where the OVA file will be created.  
 
-The following figure depicts the process of exporting a VM with a single disk as OVA:  
+The following figure depicts the process of exporting a VM with a single disk as an OVA file:  
 
 ![](../../../../images/wiki/export-ova.png)  
 
-1. The front-end sends the back-end a VM and a path on a host.
-2. The back-end asks imagio-proxy to initialize a file at the given path with an estimated size of the produced OVA.
+1. The front end sends the back end a VM and a path on a host.
+2. The back end asks imagio-proxy to initialize a file at the given path with an estimated size of the produced OVA file.
 3. Imageio creates the OVA file.
-4. The back-end sends imageio-proxy the disk to stream, which sends it to imageio.
+4. The disk is streamed from the back end to imageio-proxy, and then to imageio.
 5. Imageio reads the disk and streams it into a QCOW volume within the OVA.
-6. The back-end sends the complete OVF specification to imageio-proxy, which sends it to imageio that streams it into the OVA.
+6. The complete OVF specification is sent from the back end to imageio-proxy. imageio-proxy sends the specification to imageio, and imageio streams it into the OVA file.  
 
 Notes:
 
-1. Step 5 is equivalent to do qemu-img convert with the collapse option and the target format being QCOW, on-the-fly.
-2. The streaming in step 5 should be done in a certain offset so the streaming of the OVA would be done into the beginning of the file (the OVF should be the first tar entry inside the OVA according to the OVA specification).
-3. The back-end should receive the overall amount of data written to the disk entry in step 4 so it could be placed inside the OVF.  
-4. The OVA file will be created with a special extension (step 3) and would be renamed (that extension would be dropped) at the end of the process (step 6). That is done in order to enable tools that synchronize shares between various deployments differentiate partial files and complete files.  
+1. Step 5 is equivalent to `qemu-img convert` with the collapse option set and the target format set to QCOW.  
+2. The OVF configuration needs to be streamed to the beginning of the OVA file (according to the OVA specification, the OVF should be the first tar entry inside the OVA file).  
+3. The back end should be provided with the overall amount of data written to the disk entry in step 4, because this information needs to be set inside the OVF configuration.  
+4. The OVA file will be created with a special extension (step 3) and would be renamed (that extension would be dropped) at the end of the process (step 6). That is done in order to enable tools that synchronize shares between various deployments to differentiate partial and complete files.  
```

Comment:
@ahadas Suggest "The OVA file will be created with a special extension (step 3). At the end of the process (step 6), the the file will be renamed, and the extension dropped.